vyper/parser/expr.py

```diff
@@ -982,9 +983,16 @@ def unary_operations(self):
                 num.end_col_offset = self.expr.end_col_offset
                 return Expr.parse_value_expr(num, self.context)
 
-            return LLLnode.from_list(["sub", 0, operand], typ=operand.typ, pos=getpos(self.expr))
+            # Clamp on minimum integer value as we cannot negate that value
+            # (all other integer values are fine)
+            min_int_val, _, _ = BUILTIN_CONSTANTS['MIN_'+operand.typ.typ.upper()]
```

Comment:
I think for maintainability we should add a function in `types.py` or `expr.py` which is like `get_min_for_type()` instead of relying on the keys of BUILTIN_CONSTANTS to follow a certain rule vis a vis the type names.

Reply:
yeah, I was thinking about that